Patch-Source: https://gitlab.com/ubports/development/core/libusermetrics/-/commit/073b88128a3fd8f3333cae2ba933f29ea6e7a27e
--
From 073b88128a3fd8f3333cae2ba933f29ea6e7a27e Mon Sep 17 00:00:00 2001
From: Marius <mariogrip@debian.org>
Date: Thu, 1 Feb 2024 03:35:14 +0100
Subject: [PATCH] tests: Use en_US.UTF-8 as gettext will ignore LANGUAGE in
 newer gettext versions

See: https://github.com/autotools-mirror/gettext/commit/3873b7f1c777e6b2c874ef7c8da3f66f3b02e3fc

[ratchanan@ubports.com:
 - Taken from Debian packaging.
 - Fix misspelling in the commit subject.
]
---
 tests/testutils/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/testutils/main.cpp b/tests/testutils/main.cpp
index b9f2ce6..2a176e4 100644
--- a/tests/testutils/main.cpp
+++ b/tests/testutils/main.cpp
@@ -22,7 +22,7 @@
 #include <libintl.h>
 
 int main(int argc, char **argv) {
-	setenv("LANG", "C.UTF-8", true);
+	setenv("LANG", "en_US.UTF-8", true);
 	unsetenv("LC_ALL");
 
 	setlocale(LC_ALL, "");
-- 
GitLab

